assets/differentialabundance_report.Rmd

```diff
@@ -425,13 +427,22 @@ Input was a matrix of `r nrow(assay_data$raw)` `r params$features_type`s for `r
 
 ## Exploratory analysis
 
+```{r, echo=FALSE, results='asis', fig.height=8, eval=(!is.null(params$features_biotype_col) & !is.null(params$features))}cat("### Number of differentially expressed genes by gene type")
```

Comment:
```suggestion
```{r, echo=FALSE, results='asis', fig.height=8, eval=(!is.null(params$features_biotype_col) & !is.null(params$features))}
cat("### Number of differentially expressed genes by gene type")
```